src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -68,6 +68,29 @@ class CBSpriteNode: SKSpriteNode {
     }
 
     @objc func update(_ currentTime: TimeInterval) {
+
+        guard let previousPosition = penConfiguration.previousPosition else {
+            penConfiguration.previousPosition = self.catrobatPosition
+            return
+        }
+
+        if (penConfiguration.penDown) && (previousPosition != self.catrobatPosition) {
+
+            let path = CGMutablePath()
+            path.move(to: previousPosition)
+            path.addLine(to: self.position)
```

Comment:
It looks like you are mixing the user coordinate and the SpriteKit coordinate system. I would suggest to always use `self.position`.